src/test/java/it/pagopa/pn/cucumber/steps/SharedSteps.java

```diff
@@ -548,7 +548,7 @@ public String getFirstGroupByPa(String settedPa) {
         return id;
     }
 
```

Comment:
Unirei i metodi "last" e "first" in un unico metodo, che riceve in ingresso magari un'enum per evitare la duplicazione del codice